dotcom-rendering/src/web/components/Callout/MessageUs.tsx

```diff
@@ -0,0 +1,98 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import type { ArticleFormat } from '@guardian/libs';
+import { neutral, space, textSans } from '@guardian/source-foundations';
+import {
+	LinkButton,
+	SvgGps,
+	SvgWhatsApp,
+} from '@guardian/source-react-components';
+import { decidePalette } from '../../lib/decidePalette';
+
+interface CalloutContactProps {
+	format: ArticleFormat;
+}
+
+const calloutPrimaryButton = (format: ArticleFormat): SerializedStyles => css`
+	background: ${decidePalette(format).text.calloutAccent};
+	color: ${neutral[100]} !important;
```

Comment:
Had to add this back in otherwise the button didn't format correctly. @sophie-macmillan if you want to pair on a better solution to this please let me know.

Reply:
This fixes my button, but the linter doesn't like it. May need another solution after all @sophie-macmillan